src/algebra/big_operators/finprod.lean

```diff
@@ -416,10 +414,8 @@ by simp [h] {contextual := tt}
 ### Distributivity w.r.t. addition, subtraction, and (scalar) multiplication
 -/
 
-/-- If the multiplicative supports of `f` and `g` are finite, then the product of `f i * g i` equals
-the product of `f i` multiplied by the product over `g i`. -/
-@[to_additive] lemma finprod_mul_distrib (hf : (mul_support f).finite)
-  (hg : (mul_support g).finite) :
+@[to_additive]
+lemma finprod_mul_distrib (hf : (mul_support f).finite) (hg : (mul_support g).finite) :
```

Comment:
What's the logic behind randomly deleting perfectly good docstrings?

Reply:
The logic is that I got bored of copying lemma statements over 30 times. Do you really think those docstrings add any value?

Reply:
I also would delete a doc-string only if it is wrong, not uninformative.  In fact, whenever there is a doc-string, I always read it first and I prefer to have a quick description in words, before parsing the actual Lean code.

Reply:
I also would delete (or better correct) a doc-string only if it is wrong, not uninformative.  In fact, whenever there is a doc-string, I always read it first and I prefer to have a quick description in words, before parsing the actual Lean code.

Reply:
Yes, I agree. Even doc-strings that just repeat the statement "in English" are valuable.

Reply:
My motto is that a bad docstring is worse than no docstring. And this to me is a bad docstring: It takes more time to read it than to parse the actual statement, thus diverting your attention from what's actually important. Not to mention that it inflates the file to an unreasonable size without justification.

But sure. I had planned on such a rebutal, so I have an intermediate commit with everything you ask for.